tools/db/migrate-sso-feature-flag/src/Work.hs

```diff
@@ -66,6 +66,7 @@ writeSsoFlags = mapM_ (`setSSOTeamConfig` FeatureStatusEnabled)
   where
     setSSOTeamConfig :: MonadClient m => TeamId -> FeatureStatus -> m ()
     setSSOTeamConfig tid ssoTeamConfigStatus = do
-      retry x5 $ write updateSSOTeamConfig (params LocalQuorum (ssoTeamConfigStatus, tid))
-    updateSSOTeamConfig :: PrepQuery W (FeatureStatus, TeamId) ()
-    updateSSOTeamConfig = "update team_features set sso_status = ? where team_id = ?"
+      retry x5 . void $ trans updateSSOTeamConfig $ params LocalQuorum (ssoTeamConfigStatus, tid)
```

Comment:
Not sure here either?

Reply:
I'm pretty sure we don't want to configure deleted or never-created teams, so I think this is fine.  But if you don't believe me I'll change it back, this is almost-dead code anyway.  :)